src/mongo/db/storage/wiredtiger/SConscript

```diff
@@ -60,7 +60,6 @@ wtEnv.Library(
         '$BUILD_DIR/mongo/db/concurrency/write_conflict_exception',
         '$BUILD_DIR/mongo/db/curop',
         '$BUILD_DIR/mongo/db/encryption/encryption_kmip',
```

Comment:
I would expect that if we can remove "encryption_vault" here then we can also remove "encryption_kmip"?

Reply:
The snippet is outdated, 'encrypton_kmip' has been removed in the most recent version of the source code